draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2505,14 +2316,10 @@ to full implementations.
 
 <section anchor="sec-role-conflict" title="Detecting and Repairing Role Conflicts">
 <t>
-Normally, the rules for selection of a role in <xref target="sec-role"/>
-will result in each agent selecting a different role -- one controlling
-and one controlled. However, in unusual call flows, typically utilizing
-third party call control, it is possible for both agents to select the
-same role. This section describes procedures for checking for this case
-and repairing it. These procedures apply only to usages of ICE that
-require conflict resolution. The usage document MUST specify whether this
-mechanism is needed.
```

Comment:
Yeah, I thought "certain usages" is sufficient.  Why both with an unusual example of an unusual call flow?